tests/libmemcached-1.0/all_tests.cc

```diff
@@ -71,7 +71,7 @@
 
 #include <algorithm>
 
```

Comment:
During linking of the test cases and prod files was running into the error below. I had renamed a couple methods that contained the same signature causing multiple definition errors. 

`  CXXLD    tests/libmemcached-1.0/testapp
tests/libmemcached-1.0/tests_libmemcached_1_0_testapp-dynamic_mode_test.o: In function `get_world':
/usr/include/c++/4.8/sstream:129: multiple definition of `get_world'
tests/libmemcached-1.0/tests_libmemcached_1_0_testapp-all_tests.o:/home/ubuntu/new/aws-elasticache-cluster-client-libmemcached/BUILD/../tests/libmemcached-1.0/all_tests.cc:75: first defined here
tests/libmemcached-1.0/tests_libmemcached_1_0_testapp-dynamic_mode_test.o:(.data.rel+0x0): multiple definition of `collection'
tests/libmemcached-1.0/tests_libmemcached_1_0_testapp-all_tests.o:(.data.rel+0x0): first defined here
/usr/bin/ld: Warning: size of symbol `collection' changed from 2272 in tests/libmemcached-1.0/tests_libmemcached_1_0_testapp-all_tests.o to 64 in tests/libmemcached-1.0/tests_libmemcached_1_0_testapp-dynamic_mode_test.o
collect2: error: ld returned 1 exit status
`